mysql-test/suite/rpl/t/rpl_killed_ddl.test

```diff
@@ -162,12 +162,12 @@ let $rpl_diff_statement= SELECT event_name, event_body, execute_at
                          ORDER BY event_name;
 
 send CREATE EVENT e2
-  ON SCHEDULE AT CURRENT_TIMESTAMP + INTERVAL 1 DAY
+  ON SCHEDULE AT '2020-01-01 00:00:00' + INTERVAL 1 DAY
```

Comment:
As far as I remember, events cannot be created in the past. So, my concern is that in 3 years this test  will be failing.